partitioned-table.md

```diff
@@ -1662,6 +1662,92 @@ CREATE TABLE t (a varchar(20), b blob,
 ERROR 1503 (HY000): A UNIQUE INDEX must include all columns in the table's partitioning function
 ```
 
+#### Global indexes
+
```

Comment:
### Global indexes
Before the introduction of Global Indexes, TiDB created a Local Index for each partition, meaning one Local Index per partition. Therefore, Primary Keys and Unique Keys had to include the partition key to ensure uniqueness of the data. Additionally, when querying data that spans multiple partitions, the system needed to scan each partition’s data to return results.

To address these issues, TiDB introduced Global Indexes starting from version 8.3. Global Indexes cover the entire table's data with a single index, allowing Primary Keys and Unique Keys to maintain global uniqueness without including the partition key. At the same time, Global Indexes can access data spanning multiple partitions in a single operation, significantly improving query performance for non-partitioned keys.